lib/delivery.server.js

```diff
@@ -146,7 +152,7 @@ Delivery.prototype.subscribe = function() {
 
   //server sending to client
   this.socket.on('send.success', function(uid){
-    _this.pubSub.publish('send.success', uid);
+    _this.pubSub.publish('receive.success', uid);
```

Comment:
Do you mind testing the scenario of sending from a browser to a server? If that works as well then I'm happy to merge this. Thanks for your hard work!